src/Uno.UWP/ApplicationModel/Core/CoreApplicationView.cs

```diff
@@ -10,6 +10,8 @@ public CoreApplicationView()
 		{
 		}
 
+		public CoreWindow CoreWindow => CoreWindow.Main;
```

Comment:
Codacy found an issue: [Add a way to break out of this property's recursion.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6206176)